views/employees.py

```diff
@@ -102,10 +102,14 @@ def employee_uri():
             if empl_id:
                 for x in empl_id:
```

Comment:
yo i didn't make that name and ill delete that function what would you like that name to be?